bokeh/charts/gmap.py

```diff
@@ -24,7 +24,7 @@
 #-----------------------------------------------------------------------------
 
 
-class GMap(ChartObject):
+class GMap(Builder):
```

Comment:
Typical pattern would have `Chart` inspect any added builders for some sort of metadata or tag. If any builder has the tag for gmap plot, then `Chart` will create a `GMapPlot` or replace an existing plot with a `GMapPlot` (copying all the renderers into the new `GMapPlot`)